compiler/rustc_session/src/session.rs

```diff
@@ -196,7 +190,7 @@ pub struct Session {
     /// Tracks the current behavior of the CTFE engine when an error occurs.
     /// Options range from returning the error without a backtrace to returning an error
     /// and immediately printing the backtrace to stderr.
-    pub ctfe_backtrace: Lock<CtfeBacktrace>,
```

Comment:
This is modified by Miri

Reply:
I see. Removed this commit.

Reply:
Good catch, @Aaron1011 !

Might be good to add a comment explaining this in the code.

Reply:
Added